app/components/wds-global-footer.js

```diff
@@ -1,6 +1,15 @@
 import Component from '@ember/component';
+import { assert } from '@ember/debug';
+import track from '../utils/track';
 
 export default Component.extend({
 	tagName: 'footer',
 	classNames: 'wds-global-footer',
+
+	click(event) {
+		if (this.track) {
```

Comment:
should we add assertion for track?

Reply:
Nah, if anything then I'd add a default `track: () => {}`